docs/docusaurus/docs/core/introduction/community_resources.md

```diff
@@ -1,8 +1,22 @@
 ---
-title: Contribute
-id: contributing
+title: 'Community Resources'
 ---
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
 
+Great Expectations (GX) is committed to supporting and growing the GX community. It’s not enough to build a great tool. GX wants to build a great community as well.
+
+Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and GX is dedicated to making you feel welcome.
+
+## Contact Great Expectations
+Join the GX [public Slack channel](https://greatexpectations.io/slack). Before you post for the first time, review the [Slack Guidelines](https://discourse.greatexpectations.io/t/slack-guidelines/1195).
+
+## Ask a question
+Slack is good for that, too: [join Slack](https://greatexpectations.io/slack) and read [How to write a good question in Slack](https://github.com/great-expectations/great_expectations/discussions/4951). You can also use [GitHub Discussions](https://github.com/great-expectations/great_expectations/discussions/4951).
```

Comment:
We should replace references to Slack when asking a question with references to Discourse instead. There is also a section on "How to get your questions answered" within Discourse as well: https://discourse.greatexpectations.io/t/how-to-get-your-questions-answered/1191

Reply:
Yes!! Let's remove all mention of Slack in the context of asking questions or getting support.